include/c/sk_types.h

```diff
@@ -697,6 +697,34 @@ typedef struct {
     unsigned int fFormat;
 } gr_gl_framebufferinfo_t;
 
+typedef struct gr_vkbackendcontext_t gr_vkbackendcontext_t;
+typedef struct gr_vkinterface_t gr_vkinterface_t;
+
+typedef struct {
+    uint64_t  fMemory;
+    uint64_t  fOffset;
+    uint64_t  fSize;
+    uint32_t  fFlags;
+    bool      _private_fUsesSystemHeap;
```

Comment:
The private members are also important as the size of the struct needs to match on both the managed and the native sides.